src/config.hpp.in

```diff
@@ -25,6 +25,7 @@
 #cmakedefine ENABLE_MKLCPU_BACKEND
 #cmakedefine ENABLE_MKLGPU_BACKEND
 #cmakedefine ENABLE_NETLIB_BACKEND
+#cmakedefine DISABLE_HALF_RUTINES
```

Comment:
typo: `routines` and also I'm not really a fan of disabling stuff... can we make this a `ENABLE..` and default to `ON` or isn't it possible to default something in this weird file? :)